samvad/ner/_cast.py

```diff
@@ -0,0 +1,8 @@
+from samvad.core import casting, presets
+
+from ._step import PerformNEROutput
+
+
+class CastNEROutputToText(casting.CastOutputToContext):
+    def cast(self, context: presets.StringGeneralContext, output: PerformNEROutput) -> presets.StringGeneralContext:
+        return presets.StringGeneralOutput("Not Implemented...")
```

Comment:
@devshahofficial this is where you have to write casting logic.
fill method `cast`. It receives 2 params,
context: (Context obj which was passed to previous step, perform_ner in our case)
output: Output generated by perform_ner.

Use these to parameters to create context(input) for next step and return it